passwordGenerator.js

```diff
@@ -0,0 +1,70 @@
+var generate = document.getElementById('generator'); 
+var passInput = document.getElementById('password');
+
+generate.addEventListener("click", function() {
+    var password = createPassword(); // saving the password.
+    passInput.value = password; // changing the original value to the new password.
+});
+
+function createPassword() {
+    var newPassword = randomLowerCase(3);
+    newPassword += randomUpperCase(3);
+    newPassword += randomNumber(1);
+    newPassword += randomSpecial(1);
+    return scramble(newPassword);
+}
+
+// Math.floor = integer.
+// Math.random = generate an random float between 0 and 1, that's why we need the m.floor.
+// *26 = alphabet.
+function randomLowerCase(length) {
```

Comment:
I see a repeated pattern in your code. you always declare the result, declare the initial random list, you randomize based on the size of the list and the length then return. you can create a generic function to do this ;)